pkg/kubelet/logs/container_log_manager.go

```diff
@@ -419,7 +419,7 @@ func (c *containerLogManager) compressLog(log string) error {
 	}
 	defer r.Close()
 	tmpLog := log + tmpSuffix
-	f, err := c.osInterface.OpenFile(tmpLog, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0644)
+	f, err := c.osInterface.OpenFile(tmpLog, os.O_WRONLY|os.O_CREATE|os.O_TRUNC, 0600)
```

Comment:
Would it make sense to use the same permissions for the uncompressed and compressed logs? In this case it means the code should get permissions from the uncompressed log rather than use hardcoded `0600`.

Reply:
looks better, thanks!

/ok-to-test
/triage accepted